kolibri/core/analytics/management/commands/benchmark.py

```diff
@@ -0,0 +1,137 @@
+import sys
+
+from django.conf import settings
+from django.core.management.base import BaseCommand
+from morango.models import InstanceIDModel
+
+import kolibri
+from kolibri.core.analytics.measurements import get_channels_usage_info
+from kolibri.core.analytics.measurements import get_db_info
+from kolibri.core.analytics.measurements import get_kolibri_process_cmd
+from kolibri.core.analytics.measurements import get_kolibri_use
+from kolibri.core.analytics.measurements import get_machine_info
+from kolibri.core.analytics.measurements import get_requests_info
+from kolibri.core.analytics.measurements import KolibriNotRunning
+from kolibri.core.analytics.pskolibri.common import LINUX
+from kolibri.core.analytics.pskolibri.common import WINDOWS
+from kolibri.utils.server import installation_type
+from kolibri.utils.system import get_free_space
+from kolibri.utils.time import local_now
+
+
+def format_line(parameter, value, indented=False):
+    if indented:
+        info = '  * {:30}'.format('{}:'.format(parameter))
+    else:
+        info = '* {:32}'.format('{}:'.format(parameter))
+    return '{info}{value}'.format(info=info, value=value)
+
+
+class Command(BaseCommand):
+    """
+    This command will output information about different parameters of the server running Kolibri
+    Output example:
+
+    Sessions
+    * Active sessions (guests incl): 10
+    * Active users in (10 min):      6
+    * Active users in (1 min):       3
+
+    CPU
+    * Total processes:               351
+    * Used CPU:                      33.6%
+    * Kolibri CPU usage:             22.3%
+
+    Memory
+    * Used memory:                   9.3 GB
+    * Total memory:                  16.0 GB
+    * Kolibri memory usage:          56.8 MB
+
+    Channels
+    * Total Channels:                2
+    * Khan Academy (English)
+      * Accesses:                    150
+      * Time spent:                  301.22 s
+    * African Storybook
+      * Accesses:                     3
+      * Time spent:                  18.00 s
+
+    Requests timing
+    * Homepage:                       0.03 s
+    * Recommended channels:           0.01 s
+    * Channels:                       0.02 s
+
+    Device info
+    * Version:                       (version)
+    * OS:                            (os)
+    * Installer:                     (installer)
+    * Database:                      (database_path)
+    * Device name:                   (device_name)
+    * Free disk space:               (content_storage_free_space)
+    * Server time:                   (server_time)
+    * Server timezone:               (server_timezone)
+    """
+    help = "Outputs performance info and statistics of usage for the running Kolibri instance in this server"
+
+    def handle(self, *args, **options):
+        if not LINUX and not WINDOWS:
+            print("This OS is not yet supported")
+            sys.exit(0)
```

Comment:
I think the exit code should be non-zero?

Reply:
Honestly, I also thought it, but then I thought that not being able to use it is not an error... I don't have a strong opinion, both values could be valid. I will change it happily if it's the common opinion.

Reply:
I'll gladly have that opinion to make this easier to decide :)

Let's return non-zero... zero should indicate that a command was successfully completed... not being able to run the command is not quite successful, although the program terminates without exceptions. We had some unanticipated events in the past because of this exact kind of assumption.. just can't recall what it was.

Reply:
Here's one.. but am pretty sure there's been another recent case... https://github.com/learningequality/kolibri/pull/2881

Reply:
Perfect, changed